test/unit_test/test_similarity.py

```diff
@@ -1,80 +1,74 @@
-                         0.0],
-                        ['catCaterpillar', 0.0, 0.0, 9.0, 9.0, 0.0, 0.0, 0.0,
-                         4.0, 5.0, 9.0, 0.0, 0.0, 0.0, 0.0, 0.0, 9.0, 0.0, 0.0,
-                         0.0],
-                        ['wake', 5.0, 10.0, 0.0, 0.0, 10.0, 5.0, 0.0, 0.0, 0.0,
-                         0.0, 10.0, 5.0, 5.0, 5.0, 5.0, 0.0, 5.0, 5.0, 5.0]]
+        count_matrix = np.array([[0.0, 0.0, 9.0, 9.0, 0.0, 0.0, 5.0, 4.0,
+                                  0.0, 9.0, 0.0, 0.0, 0.0, 0.0, 0.0, 9.0, 0.0,
+                                  0.0, 0.0],
+                                 [0.0, 0.0, 9.0, 9.0, 0.0, 0.0, 0.0, 4.0, 5.0,
+                                  9.0, 0.0, 0.0, 0.0, 0.0, 0.0, 9.0, 0.0, 0.0,
+                                  0.0],
+                                 [5.0, 10.0, 0.0, 0.0, 10.0, 5.0, 0.0, 0.0,
+                                  0.0, 0.0, 10.0, 5.0, 5.0, 5.0, 5.0, 0.0, 5.0,
+                                  5.0, 5.0]], dtype=object)
         comp_file_index = 2
-        temp_labels = ['catBobcat', 'catCaterpillar']
-        assert similarity_maker(count_matrix, comp_file_index, temp_labels
-                                ) == ([1.0, 1.0], ['catBobcat',
-                                                   'catCaterpillar'])
+        temp_labels = np.array(['catBobcat', 'catCaterpillar'])
+        np.testing.assert_equal(
+            similarity_maker(count_matrix, comp_file_index, temp_labels),
+            (np.array([1.0, 1.0]), np.array(['catBobcat', 'catCaterpillar']))
+        )
 
     def test_with_all_same_content_file(self):
-        count_matrix = [['', 'The', 'all', 'bobcat', 'cat', 'caterpillar',
-                         'day.', 'slept'],
-                        ['catBobcat', 9.0, 9.0, 5.0, 4.0, 0.0, 9.0, 9.0],
-                        ['catCaterpillar', 9.0, 9.0, 5.0, 4.0, 0.0, 9.0, 9.0],
-                        ['test', 9.0, 9.0, 5.0, 4.0, 0.0, 9.0, 9.0]]
+        count_matrix = np.array([[9.0, 9.0, 5.0, 4.0, 0.0, 9.0, 9.0],
+                                [9.0, 9.0, 5.0, 4.0, 0.0, 9.0, 9.0],
+                                [9.0, 9.0, 5.0, 4.0, 0.0, 9.0, 9.0]],
+                                dtype=object)
         comp_file_index = 1
-        temp_labels = ['catBobcat', 'test']
-        assert similarity_maker(count_matrix, comp_file_index, temp_labels
-                                ) == ([0.0, 0.0], ['catBobcat', 'test'])
+        temp_labels = np.array(['catBobcat', 'test'])
+        np.testing.assert_equal(
+            similarity_maker(count_matrix, comp_file_index, temp_labels),
+            (np.array([0.0, 0.0]), np.array(['catBobcat', 'test']))
+        )
 
     def test_with_two_dimension(self):
-        count_matrix = [['', 'cat', 'the'], ['file1', 0.0, 1.0],
-                        ['file2', 1.0, 2.0], ['file3', 2.0, 1.0]]
+        count_matrix = np.array([[0.0, 1.0], [1.0, 2.0],
+                                 [2.0, 1.0]], dtype=object)
```

Comment:
dtype should not be object, `double` or `float` probably (I am not sure python has `double` type)